src/components/InputProblem.tsx

```diff
@@ -132,12 +134,26 @@ export const InputProblem = ({
                     <WrongAnswerIcon className={'os-mr'} />
                   </div>
                 }
-                <Field
-                name="response"
-                disabled={inputDisabled || isSubmitting}
-                autoComplete={'off'}
-                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => { clearFeedback(); void setFieldValue('response', e.target.value) }}
-                className={buildClassName(values.response, solution, inputDisabled || isSubmitting)} />
+              {
+                comparator.toLowerCase() === 'math'
+                  ? (
+                  <Field
+                    name="response"
+                    disabled={inputDisabled}
+                    as={Mathfield}
+                    onInput={(e: React.ChangeEvent<MathfieldElement>): void => { clearFeedback(); void setFieldValue('response', e.target.value); console.log('This is the value:', e.target.value) }}
```

Comment:
I think this has a debug `console.log` which we probably want to remove.

Reply:
Fixed in latest PR.